src/Mod/Spreadsheet/Gui/Command.cpp

```diff
@@ -393,8 +393,8 @@ CmdSpreadsheetAlignRight::CmdSpreadsheetAlignRight()
 {
     sAppModule = "Spreadsheet";
     sGroup = QT_TR_NOOP("Spreadsheet");
-    sMenuText = QT_TR_NOOP("Align &right");
-    sToolTipText = QT_TR_NOOP("Right-align contents of selected cells");
+    sMenuText = QT_TR_NOOP("Align &Right");

```

Comment:
```suggestion
    sMenuText = QT_TR_NOOP("&Right Align");
```
Pending Discord discussion